main.go

```diff
@@ -76,26 +76,26 @@ func main() {
 
 	image, err := docker.NewImage(os.Args[1], dockerUser, dockerPassword, insecureTLS, insecureRegistry)
 	if err != nil {
-		fmt.Printf("Can't parse qname: %s", err)
+		fmt.Fprintf(os.Stderr, "Can't parse qname: %s", err)
 		os.Exit(1)
 	}
 
 	err = image.Pull()
 	if err != nil {
-		fmt.Printf("Can't pull image: %s", err)
+		fmt.Fprintf(os.Stderr, "Can't pull image: %s", err)
 		os.Exit(1)
 	}
 
 	var output = jsonOutput{}
 
 	if len(image.FsLayers) == 0 {
-		fmt.Printf("Can't pull fsLayers")
+		fmt.Fprintf(os.Stderr, "Can't pull fsLayers")
 		os.Exit(1)
 	} else {
 		if useJSONOutput {
 			output.LayerCount = len(image.FsLayers)
 		} else {
-			fmt.Printf("Analysing %d layers\n", len(image.FsLayers))
+			fmt.Fprintf(os.Stderr, "Analysing %d layers\n", len(image.FsLayers))
```

Comment:
Is this still not fixed? It's not included in [the revertion commit](https://github.com/optiopay/klar/pull/29/commits/3283eee1e2e891b2721d6f08e2e96440161d4c6e).